app/javascript/packages/document-capture/components/in-person-prepare-step.jsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Alert,
+  Icon,
+  IconList,
+  IconListContent,
+  IconListIcon,
+  IconListItem,
+  IconListTitle,
+  PageHeading,
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem,
+} from '@18f/identity-components';
+import { FormStepsButton } from '@18f/identity-form-steps';
+import { useI18n } from '@18f/identity-react-i18n';
+import InPersonTroubleshootingOptions from './in-person-troubleshooting-options';
+
+/**
+ * @typedef InPersonPrepareStepValue
+ *
+ * @prop {Blob|string|null|undefined} inPersonPrepare InPersonPrepare value.
+ */
+
+/**
+ * @param {import('@18f/identity-form-steps').FormStepComponentProps<InPersonPrepareValue>} props Props object.
+ */
+function InPersonPrepareStep({ errors = [] }) {
+  const { t } = useI18n();
+  const error = errors.find(({ field }) => field === 'in-person-prepare')?.error;
+
+  return (
+    <>
+      <Alert type="success" className="margin-bottom-4">
+        {t('in_person_proofing.body.prepare.alert_selected_post_office', { name: 'EASTCHESTER' })}
+      </Alert>
+      <PageHeading>{t('in_person_proofing.headings.prepare')}</PageHeading>
+
+      <p>{t('in_person_proofing.body.prepare.verify_step_about')}</p>
+
+      <ProcessList className="margin-bottom-4">
+        <ProcessListItem>
+          <ProcessListHeading unstyled>
+            {t('in_person_proofing.body.prepare.verify_step_enter_pii')}
+          </ProcessListHeading>
+        </ProcessListItem>
+        <ProcessListItem>
+          <ProcessListHeading unstyled>
+            {t('in_person_proofing.body.prepare.verify_step_enter_phone')}
+          </ProcessListHeading>
+        </ProcessListItem>
+      </ProcessList>
+
+      <hr className="margin-bottom-4" />
+
+      <h2>{t('in_person_proofing.body.prepare.bring_title')}</h2>
+
+      <IconList>
+        <IconListItem>
+          <IconListIcon className="text-primary-dark">
+            <Icon icon="check_circle" />
+          </IconListIcon>
+          <IconListContent>
+            <IconListTitle className="font-sans-md padding-top-0">
```

Comment:
Are these common styles to all icon list icons and titles? Should they be absorbed into those components statically, so we don't have to add the props here? (Or I guess more importantly, can we avoid customizing the base design system styles or update those accordingly if we need to?)

Reply:
I moved the styles into the components statically in bcb422e1d16cca018ee90deed476b0db910ae604, and wrote LG-6873 to backlog the work of updating the base design system styles if appropriate.